playbooks/streisand.yml

```diff
@@ -40,11 +40,13 @@
     - role: wireguard
       when: streisand_wireguard_enabled
     - role: cloudflared
-      when: streisand_cloudflared_enabled
+      when: streisand_cloudflared_enabled and not streisand_ci
     # streisand_le_enabled is set in lets-encrypt.yml based on user input.
     # lets-encrypt roles sets le_ok, which is used by streisand-gateway.
     - role: lets-encrypt
       when: streisand_le_enabled
+    - role: ad-blocking
```

Comment:
I think you need to either rebase this commit on top of https://github.com/StreisandEffect/streisand/pull/1608 or remove reference to the `ad-blocking` role in this commit. CI is failing trying to find the `ad-blocking` role

Reply:
ooops